xtask/codegen/src/css_kinds_src.rs

```diff
@@ -92,6 +92,19 @@ pub const CSS_KINDS_SRC: KindsSrc = KindsSrc {
         "url",
         "font-palette-values",
         "auto",
+        "thin",
+        "medium",
+        "thick",
+        "none",
+        "hidden",
+        "dotted",
+        "dashed",
+        "solid",
+        "double",
+        "groove",
+        "ridge",
+        "inset",
+        "outset",
```

Comment:
This is just a single property....and there are plenty with even more keywords. I think it might be worth just treating these as plain identifiers rather than expanding this list to like 1000 keywords, but I'm not really sure. Both ways have benefits and drawbacks.

Reply:
I agree, I don't have a certain opinion about it.

I like having them in the ungram to see which keywords I can use, and to assert in a factory if one wants to use an invalid keyword. Another thing to consider is that implementing a binary search in the lexer could offer a performance benefit, as it would allow us to compare enum variants instead of strings.